### DISCUSSION TITLE
How to contribute a theme?

### DISCUSSION THREAD

**Discussion by neonvoidx** [wezterm/wezterm#5186] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows, Linux Wayland, macOS

### Which Wayland compositor or X11 Window manager(s) are you using?

N/A

### WezTerm version

N/A

### Ask your question!

I was wondering how I can contribute a theme I made thats in .toml format?

**Comment by neonvoidx** [comment]
@wez ?

**Reply by wez** [reply, marked as answer]
The schemes are sync'd from upstream repos.
If you have one of your own that you'd like to add, take a look at https://github.com/EdenEast/nightfox.nvim/blob/main/extra/dayfox/wezterm.toml
for an example of the toml file and metadata.
To get it added, you'd submit a PR here to wezterm to add a few lines like this, but with your repo listed instead:
https://github.com/wez/wezterm/blob/e5ac32f297cf3dd8f6ea280c130103f3cac4dddb/sync-color-schemes/src/main.rs#L384-L386

**Reply by neonvoidx** [reply]
ok thank you https://github.com/wez/wezterm/pull/5232 submitted

**Reply by neonvoidx** [reply]
Do i need to add a color printout screenshot? If so whats the command to generate that to look like the rest of the themes in the Configuration Color Schemes list on documentation website?

**Reply by wez** [reply]
I run `cargo run -p sync-color-schemes` to update the various data files, then `./ci/build-docs.sh serve` to build and run the docs locally; it should automatically add an entry with your scheme.  That script will print our the URL that you can use to visit your instance of the docs in your browser.

Please do not include the changes made from running `sync-color-schemes` in your commit/PR, as there will likely be some other changes for me to reconcile from other theme changes.

**Reply by neonvoidx** [reply]
after running `.ci/build-docs.sh serve` i get 
![image](https://github.com/wez/wezterm/assets/25580051/be64e35e-f409-4222-8cf1-de0de4c6818a)

What am i missing?
